lib/publiccloud/instance.pm

```diff
@@ -848,5 +848,25 @@ sub do_systemd_analyze_time {
     return @ret;
 }
 
+sub retry_on_script_output {
+    my ($self, %args) = @_;
+    my $retries = 10;
+    my $delay_command = 20;
```

Comment:
Is this in seconds?  Then I think 20 is a bit too much.  Why not try with a lower value using exponential backoff?